graphicalquerybuilder/moc/moc_predefs.h

```diff
@@ -0,0 +1,390 @@
+#define __SSP_STRONG__ 3
```

Comment:
This file don't need to be commited too... actually, all the moc/* :)

Reply:
yes I know!! damn .gitignores